README.md

```diff
@@ -64,223 +64,101 @@ createCompilerHost := (options: CompilerOptions, moduleSearchLocations : string[
-- `else` → `default` in `switch`
-- Range literals `[0...10]`, `[a..b]`, `[x - 2 .. x + 2]`
-- Array slices `list[0...2]` → `list.slice(0, 2)`
-- Slice assignment `numbers[3..6] = [-3, -4, -5, -6]` → `numbers.splice(3, 4, ...[-3, -4, -5, -6])`
-- Implicit returns
-- Late assignment `x + y = z` → `x + (y = z)`
-- Braceless inline objects `x = coolStory: true`
-- Simplified number method calls `1.toFixed()` → `1..toFixed()`
-- `if`/`switch`/`for`/`loop`/`while`/`throw` expressions
-- Destructuring object assignment doesn't require being wrapped in parens at the statement level `{a, b} = c` → `({a, b} = c)`
-- Prefix or postfix rest/splats `[...a]`, `x = [a...]`
-- RestProperty in any position `{a, ...b, c} = d` → `{a, c, ...b} = d`
-- RestElement/RestParameter in any position `(first, ...midle, last) ->` → `function(first, ...middle) { let [last] = middle.splice(-1)}`
-- `///` Heregexp
-  - With some [changes](#things-changed-from-coffeescript).
-- JSX
-
-Things Removed from CoffeeScript
----
+Civet is essentially a tasteful superset of TypeScript.
 
-Most of these can be enabled by adding a [`"civet coffeeCompat"` directive prologue](#coffeescript-compatibility) to the top of your file.
-The goal is to provide a very high level of compatibility with existing CoffeeScript code while offering a fine grained migration path to modern
-Civet.
-
-- Implicit `var` declarations (use `"civet coffeeCompat"` or `"civet autoVar"`)
-- `on/yes/off/no` (use `true/false`, `"civet coffeeCompat"`, or `"civet coffeeBooleans"` to add them back)
-- `not` (use `!`, `"civet coffeeCompat"`, or `"civet coffeeNot"`)
-  - `not instanceof` (use `!(a instanceof b)`, `"civet coffeeCompat"`, or `"civet coffeeNot"`)
-  - `not of` use (`"civet coffeeCompat"`, or `"civet coffeeNot"`)
-  - NOTE: CoffeeScript `not` precedence is dubious. `not a < b` should be equivalent to `!(a < b)` but it is in fact `!a < b`
-- `do` keyword (replaced with JS `do`, invoke using existing `(-> ...)()` syntax, `"civet coffeeCompat"`, or `"civet coffeeDo"`)
-- `for from` (use JS `for of`, `"civet coffeeCompat"`, or `"civet coffeeForLoops"`)
-- `for own of` (use JS `for in` and check manually, switch to `Map#keys/values/entries`, or use `Object.create(null)`, or `"civet coffeeCompat"`, or `"civet coffeeForLoops"`)
-- `for ... when <condition>` (use `continue if exp` inside loop, `"civet coffeeCompat"`, or `"civet coffeeForLoops"`)
-- `a ? b` (use `a ?? b`, though it doesn't check for undeclared variables; `"civet coffeeCompat"`, or `"civet coffeeBinaryExistential"` enables `a ? b` at the cost of losing JS ternary operator)
-- `a of b` (use `a in b` as in JS, or `"civet coffeeCompat"`, or `"civet coffeeOf"`)
-- Backtick embedded JS (replaced by template literals)
-- Will add later
-  - Conditional assignment `a?[x] = 3` → `a ? a[x] = 3 : undefined`
-  - Multiple slice assignment `otherNumbers[0...] = numbers[3..6] = [-3, -4, -5, -6]`
-
-Things Changed from CoffeeScript
----
+### TypeScript ehnancements
```

Comment:
Typo: `ehnancements`

Also I'd suggest moving this down lower. It's currently not a very impressive list, so let's focus on ES stuff first.

In fact, maybe the big new features section (pipe/function block) should come first, followed by conveniences?